scripts/create.sh

```diff
@@ -138,76 +148,88 @@ if [[ ! "$(kubectl get clusterrolebinding --field-selector metadata.name=cluster
 export GCP_OPTS="--zone ${ZONE} --project ${ISTIO_PROJECT}"
 export SERVICE_NAMESPACE=vm
-./install/tools/setupMeshEx.sh generateClusterEnv "${ISTIO_CLUSTER}"
+(
+  cd "$ISTIO_DIR"
+  "$ISTIO_DIR/install/tools/setupMeshEx.sh" \
+    generateClusterEnv "${ISTIO_CLUSTER}"
+)
 
 # Ensure that mTLS is enabled
 if [[ "${ISTIO_AUTH_POLICY}" == "MUTUAL_TLS" ]] ; then
-  sed -i'' -e "s/CONTROL_PLANE_AUTH_POLICY=NONE/CONTROL_PLANE_AUTH_POLICY=${ISTIO_AUTH_POLICY}/g" cluster.env
+  sed -i'' -e "s/CONTROL_PLANE_AUTH_POLICY=NONE/CONTROL_PLANE_AUTH_POLICY=${ISTIO_AUTH_POLICY}/g" "$ISTIO_DIR/cluster.env"
 fi
 
 # Generate the DNS configuration necessary to have the GCE VM join the mesh.
-./install/tools/setupMeshEx.sh generateDnsmasq
+(
+  cd "$ISTIO_DIR"
+  "$ISTIO_DIR/install/tools/setupMeshEx.sh" generateDnsmasq
+)
 
 # Create the namespace to be used by the service on the VM.
-kubectl apply -f ../namespaces.yaml
+kubectl apply -f "$ROOT/scripts/namespaces.yaml"
 
 # Create the keys for mTLS
-./install/tools/setupMeshEx.sh machineCerts default vm all
+(
+  cd "$ISTIO_DIR"
+  "$ISTIO_DIR/install/tools/setupMeshEx.sh" machineCerts default vm all
+)
 
 # Re-export the GCP_OPTS to switch the project to the project where the VM
 # resides
 export GCP_OPTS="--zone ${ZONE} --project ${GCE_PROJECT}"
 # Setup the Istio service proxy and service on the GCE VM
-./install/tools/setupMeshEx.sh gceMachineSetup "${GCE_VM}"
+(
+  cd "$ISTIO_DIR"
+  "$ISTIO_DIR/install/tools/setupMeshEx.sh" gceMachineSetup "${GCE_VM}"
+)
 
 # Mesh expansion completed
 
 # Register the external service with the Istio mesh
-istioctl register -n vm mysqldb "$(gcloud compute instances describe "${GCE_VM}" \
+"$ISTIO_DIR/bin/istioctl" register -n vm mysqldb "$(gcloud compute instances describe "${GCE_VM}" \
   --format='value(networkInterfaces[].networkIP)' --project "${GCE_PROJECT}" --zone "${ZONE}")" 3306
 
 # Install the bookinfo services and deployments and set up the initial Istio
 # routing. For more information on routing see this Istio blog post:
 # https://istio.io/blog/2018/v1alpha3-routing/
-kubectl apply -f ./samples/bookinfo/platform/kube/bookinfo.yaml
-kubectl apply -f ./samples/bookinfo/networking/bookinfo-gateway.yaml
-kubectl apply -f ./samples/bookinfo/networking/destination-rule-all-mtls.yaml
+kubectl apply -n default \
+  -f "$ISTIO_DIR/samples/bookinfo/platform/kube/bookinfo.yaml"
+kubectl apply -n default \
+  -f "$ISTIO_DIR/samples/bookinfo/networking/bookinfo-gateway.yaml"
+kubectl apply -n default \
+  -f "$ISTIO_DIR/samples/bookinfo/networking/destination-rule-all-mtls.yaml"
 
 # Change the routing to point to the most recent versions of the bookinfo
 # microservices
-kubectl apply -f ./samples/bookinfo/networking/virtual-service-reviews-v3.yaml
-kubectl apply -f ./samples/bookinfo/platform/kube/bookinfo-ratings-v2-mysql-vm.yaml
-
-popd || exit
+kubectl apply -n default \
+  -f "$ISTIO_DIR/samples/bookinfo/networking/virtual-service-reviews-v3.yaml"
```

Comment:
@samuelmi2 are these the right manifests for 1.0.1?  I thought we had to use 1.0.1 specific manifests.

Reply:
@samuelmi2 probably won't get this. This is the same configs I inherited and it was already set to 1.0.1. I did have to add the apply for `$ISTIO_DIR/samples/bookinfo/networking/virtual-service-ratings-mysql-vm.yaml` after consulting with Daniel.